### PR TITLE
Fixes and improvements

### PR DESCRIPTION
**Pull request by maximumpower55**

### CODE REVIEW

File: src/main/java/com/fusionflux/portalcubed/PortalCubed.java [hunk 1]
```diff
@@ -259,12 +255,6 @@ public void onInitialize(ModContainer mod) {
             });
         });
 
-        ServerPlayConnectionEvents.JOIN.register((handler, sender, server) -> {
-            final PacketByteBuf buf = PacketByteBufs.create();
-            buf.writeBoolean(handler.player.world.getGameRules().getBoolean(PortalCubedGameRules.ALLOW_CROUCH_FLY_GLITCH));
-            handler.sendPacket(ServerPlayNetworking.createS2CPacket(PortalCubedClientPackets.ENABLE_CFG, buf));
-        });
-
```
Comment (Gaming32) [inline src/main/java/com/fusionflux/portalcubed/PortalCubed.java]: Why'd you remove this?

File: src/main/java/com/fusionflux/portalcubed/blocks/blockentities/FloorButtonBlockEntity.java [hunk 2]
```diff
@@ -43,21 +43,24 @@ public static void tick1(World world, BlockPos pos, BlockState state, FloorButto
...
             for (LivingEntity living : entities) {
-                if (living instanceof PlayerEntity || living instanceof StorageCubeEntity || living instanceof Portal1CompanionCubeEntity || living instanceof Portal1StorageCubeEntity || living instanceof OldApCubeEntity) {
-                    isPowered = true;
-                    if (state.getBlock().equals(PortalCubedBlocks.FLOOR_BUTTON)) {
-                        if (living instanceof StorageCubeEntity cube) {
-                            cube.setButtonTimer(1);
-                        }
-                        if (living instanceof CompanionCubeEntity cube) {
-                            cube.setButtonTimer(1);
+                if (living instanceof PlayerEntity || living instanceof StorageCubeEntity || living instanceof CompanionCubeEntity || living instanceof Portal1CompanionCubeEntity || living instanceof Portal1StorageCubeEntity || living instanceof OldApCubeEntity || living instanceof RedirectionCubeEntity) {
```
Comment (Gaming32) [inline src/main/java/com/fusionflux/portalcubed/blocks/blockentities/FloorButtonBlockEntity.java]: `living instanceof CompanionCubeEntity` and `living instanceof RedirectionCubeEntity` are covered by `living instanceof StorageCubeEntity` because they're subclasses.
```suggestion
                if (living instanceof PlayerEntity || living instanceof StorageCubeEntity || living instanceof Portal1CompanionCubeEntity || living instanceof Portal1StorageCubeEntity || living instanceof OldApCubeEntity) {
```

File: src/main/java/com/fusionflux/portalcubed/blocks/fizzler/AbstractFizzlerBlock.java [hunk 3]
```diff
@@ -59,7 +59,6 @@ public static boolean isEmpty(BlockState state) {
     }
 
     @Override
-    @SuppressWarnings("deprecation")
```
Comment (Gaming32) [inline src/main/java/com/fusionflux/portalcubed/blocks/fizzler/AbstractFizzlerBlock.java]: Why'd you remove this?

File: src/main/java/com/fusionflux/portalcubed/blocks/fizzler/AbstractFizzlerBlock.java [hunk 4]
```diff
@@ -79,7 +78,6 @@ public VoxelShape getTriggerShape(BlockState state, BlockView world, BlockPos po
...
```
Comment (Gaming32) [inline src/main/java/com/fusionflux/portalcubed/blocks/fizzler/AbstractFizzlerBlock.java]: Why'd you remove this?

File: src/main/java/com/fusionflux/portalcubed/blocks/fizzler/FizzlerEmitter.java [hunk 5]
```diff
@@ -111,7 +111,6 @@ public void onPlaced(World world, BlockPos pos, BlockState state, @Nullable Livi
     }
 
     @Override
-    @SuppressWarnings("deprecation")
```
Comment (Gaming32) [inline src/main/java/com/fusionflux/portalcubed/blocks/fizzler/FizzlerEmitter.java]: Why'd you remove this?

File: src/main/java/com/fusionflux/portalcubed/client/render/block/entity/RocketTurretRenderer.java [hunk 6]
```diff
@@ -48,10 +51,8 @@ public void render(RocketTurretBlockEntity entity, float tickDelta, MatrixStack
         matrices.scale(-1, -1, 1);
         matrices.translate(0.5, -1.501, -0.5);
 
-        final BlockEntityWrapperEntity<RocketTurretBlockEntity> wrapper = new BlockEntityWrapperEntity<>(entity);
```
Comment (Gaming32) [inline src/main/java/com/fusionflux/portalcubed/client/render/block/entity/RocketTurretRenderer.java:51]: Why did you change this to use another `WeakHashMap`?

File: src/main/java/com/fusionflux/portalcubed/fluids/ToxicGooFluid.java [hunk 7]
```diff
@@ -115,7 +115,6 @@ public Block(FlowableFluid flowableFluid, Settings settings) {
         }
 
         @Override
-        @SuppressWarnings("deprecation")
```
Comment (Gaming32) [inline src/main/java/com/fusionflux/portalcubed/fluids/ToxicGooFluid.java]: Why'd you remove this?

**Review by Gaming32 — CHANGES_REQUESTED** [review]
Review go brrrr

File: src/main/java/com/fusionflux/portalcubed/client/render/block/entity/RocketTurretRenderer.java (hunk 6 above)
Comment (maximumpower55) [reply src/main/java/com/fusionflux/portalcubed/client/render/block/entity/RocketTurretRenderer.java:51]: Because this was creating a new `BlockEntityWrapperEntity` every single frame

Comment (Gaming32) [reply src/main/java/com/fusionflux/portalcubed/client/render/block/entity/RocketTurretRenderer.java:51]: That's still significantly cheaper than doing a lookup (and simpler too). Additionally, if you're arguing about allocations, that lambda will be an allocation every frame because you're capturing a local (`entity`).

File: src/main/java/com/fusionflux/portalcubed/blocks/fizzler/AbstractFizzlerBlock.java (hunk 3 above)
Comment (maximumpower55) [reply src/main/java/com/fusionflux/portalcubed/blocks/fizzler/AbstractFizzlerBlock.java]: There was no deprecation warning with it removed

File: src/main/java/com/fusionflux/portalcubed/blocks/fizzler/AbstractFizzlerBlock.java (hunk 4 above)
Comment (maximumpower55) [reply src/main/java/com/fusionflux/portalcubed/blocks/fizzler/AbstractFizzlerBlock.java]: There was no deprecation warning with it removed

File: src/main/java/com/fusionflux/portalcubed/blocks/fizzler/FizzlerEmitter.java (hunk 5 above)
Comment (maximumpower55) [reply src/main/java/com/fusionflux/portalcubed/blocks/fizzler/FizzlerEmitter.java]: There was no deprecation warning with it removed

File: src/main/java/com/fusionflux/portalcubed/fluids/ToxicGooFluid.java (hunk 7 above)
Comment (maximumpower55) [reply src/main/java/com/fusionflux/portalcubed/fluids/ToxicGooFluid.java]: There was no deprecation warning with it removed

File: src/main/java/com/fusionflux/portalcubed/blocks/fizzler/AbstractFizzlerBlock.java (hunk 3 above)
Comment (Gaming32) [reply src/main/java/com/fusionflux/portalcubed/blocks/fizzler/AbstractFizzlerBlock.java]: There's a deprecation warning in IntelliJ because you're overriding a deprecated method.

File: src/main/java/com/fusionflux/portalcubed/blocks/fizzler/AbstractFizzlerBlock.java (hunk 4 above)
Comment (Gaming32) [reply src/main/java/com/fusionflux/portalcubed/blocks/fizzler/AbstractFizzlerBlock.java]: There's a deprecation warning in IntelliJ because you're overriding a deprecated method.

File: src/main/java/com/fusionflux/portalcubed/blocks/fizzler/FizzlerEmitter.java (hunk 5 above)
Comment (Gaming32) [reply src/main/java/com/fusionflux/portalcubed/blocks/fizzler/FizzlerEmitter.java]: There's a deprecation warning in IntelliJ because you're overriding a deprecated method.

File: src/main/java/com/fusionflux/portalcubed/fluids/ToxicGooFluid.java (hunk 7 above)
Comment (Gaming32) [reply src/main/java/com/fusionflux/portalcubed/fluids/ToxicGooFluid.java]: There's a deprecation warning in IntelliJ because you're overriding a deprecated method.

File: src/main/java/com/fusionflux/portalcubed/client/render/block/entity/RocketTurretRenderer.java (hunk 6 above)
Comment (maximumpower55) [reply src/main/java/com/fusionflux/portalcubed/client/render/block/entity/RocketTurretRenderer.java:51]: I will just change it to store the block entity wrapper entity on the rocket turrent block entity

Comment (Gaming32) [reply src/main/java/com/fusionflux/portalcubed/client/render/block/entity/RocketTurretRenderer.java:51]: I think it would be better to keep the wrapper lookup and store the model on the wrapper.

Comment (maximumpower55) [reply src/main/java/com/fusionflux/portalcubed/client/render/block/entity/RocketTurretRenderer.java:51]: You would have to explain what you mean by store the model on the wrapper

Comment (Gaming32) [reply src/main/java/com/fusionflux/portalcubed/client/render/block/entity/RocketTurretRenderer.java:51]: Instead of having two separate `WeakHashMap`s for the wrappers and for the models, the models could go in the wrappers.

Comment (maximumpower55) [reply src/main/java/com/fusionflux/portalcubed/client/render/block/entity/RocketTurretRenderer.java:51]: Ah.

**Review by Gaming32 — APPROVED** [review]
LGTM
